website/content/en/highlights/2023-02-28-0-28-0-upgrade-guide.md

```diff
@@ -58,7 +58,6 @@ syntax to use to migrate.
 
```

Comment:
This (or rather `0.29.0`?) should still have a note that the old name has been deprecated and will be removed next/in a future release.